EventFilter/RawDataCollector/src/SealModule.cc

```diff
@@ -2,8 +2,10 @@
 #include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/ServiceRegistry/interface/ServiceMaker.h"
 #include "EventFilter/RawDataCollector/src/RawDataCollectorByLabel.h"
+#include "EventFilter/RawDataCollector/interface/RawDataMapperByLabel.h"
 
 using namespace edm::serviceregistry;
 
 DEFINE_FWK_MODULE(RawDataCollectorByLabel);
+DEFINE_FWK_MODULE(RawDataMapperByLabel);
```

Comment:
this line can be moved to the end of RawDataMapperByLabel.cc file

We should eventually get rid of the SealModule.cc file, but it's fine to leave this old implementation still here.

Reply:
done. I can try to remove it in the next iteration. I kept the same package structure